pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -108,6 +109,22 @@ self: super: {
     '';
   });
 
+
+  coc-nvim = let
+    version = "v0.0.67";
```

Comment:
The `v` shouldn't be part of the version.

Reply:
I will move the `v` around then xD